### PR TITLE
Zola installer 0.17.1

### PR DESCRIPTION
**Pull request by leotsarev**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

Blocked on 1.4 release

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/79060)

### CODE REVIEW

**Comment by russellbanks** [comment]
@leotsarev singleton ManifestVersion was "deprecated" a while ago; the latest is 1.2.0. To my knowledge, there isn't a single singleton manifest left in the repository

File: manifests/z/Zola/zola/0.16.1.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
```
Comment (Trenly) [inline manifests/z/Zola/zola/0.16.1.yaml:1]:
```suggestion
# ---
```

File: manifests/z/Zola/zola/0.16.1.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+PackageLocale: en-US
+Publisher: Vincent Prouillet
+License: MIT
+ShortDescription: Zola static site generator
+PackageUrl: https://www.getzola.org/
+Installers:
+  - Architecture: x64
+    InstallerUrl: https://github.com/getzola/zola/releases/download/v0.16.1/zola-v0.16.1-x86_64-pc-windows-msvc.zip
+    InstallerSha256: B19B8317A9A0D53339DF8A5D910AF0682ABAD5C65F593D58FCC35C6D00AB8732
+    InstallerType: zip
```
Comment (Trenly) [inline manifests/z/Zola/zola/0.16.1.yaml:15]: Zip type is not supported yet

File: manifests/z/Zola/zola/0.16.1.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    NestedInstallerType: portable
+    NestedInstallerFiles:
+      - RelativeFilePath: zola.exe
+        PortableCommandAlias: zola
+
+ManifestType: singleton
```
Comment (Trenly) [inline manifests/z/Zola/zola/0.16.1.yaml:21]: Please move to Multi-manifest format

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/z/Zola/zola/0.16.1.yaml (hunk 3 above)
Comment (leotsarev) [reply manifests/z/Zola/zola/0.16.1.yaml]: Thanks, fixed

File: manifests/z/Zola/zola/0.16.1.yaml (hunk 1 above)
Comment (leotsarev) [reply manifests/z/Zola/zola/0.16.1.yaml]: fixed

**Review by ItzLevvie — DISMISSED** [review]

File: manifests/z/Zola/zola/0.16.1.yaml (hunk 2 above)
Comment (leotsarev) [reply manifests/z/Zola/zola/0.16.1.yaml]: That's why this is a draft.

**Comment by ItzLevvie** [comment]
@msftbot .zip

**Review by Trenly — CHANGES_REQUESTED** [review]
Folder name `manifests/z/Zola/zola/` should be `manifests/z/Zola/Zola/`

**Comment by stephengillie** [comment]
Hi @leotsarev,

The package installs normally, but requires `Microsoft.VCRedist.2015+.x64` to run:
image

Pending:
* https://github.com/microsoft/winget-cli/issues/163

**Comment by stephengillie** [comment]
Hi @leotsarev,

Please understand that dependencies are not currently supported. Specifying the dependency in the manifest won't have it install in the pipeline nor on user computers at this time. We are hoping to have this resolved soon. It's being tracked in this issue: 
* https://github.com/microsoft/winget-cli/issues/163

**Comment by leotsarev** [comment]
Thanks @stephengillie !

I guess I have to wait yet another winget release:-)

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Review by ItzLevvie — APPROVED** [review]

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Comment by armoha** [comment]
Since packaging dependencies (https://github.com/microsoft/winget-cli/issues/163) has been supported in WinGet 1.6, can we continue to work on this issue?

**Pull request closed without merging**